lib/common/models/attribute_types/titlecase_string.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class Common::TitlecaseString < Virtus::Attribute
+  def coerce(value)
+    value&.downcase&.titlecase
+  end
+end
```

Comment:
Could we use this for the Preferred Name as well since I think that is also coming back as all caps and being transformed on the FE?

Reply:
Yeah, I don't see why not